### DISCUSSION TITLE
Two tracking blades

### DISCUSSION THREAD

**Discussion by ezitisitis** [VincentBean/laravel-plausible#21] [Q&A]
Sorry, but I just got generally confused- why there are 2 tracking blades.

https://github.com/VincentBean/laravel-plausible/blob/main/resources/views/components/tracking.blade.php
and
https://github.com/VincentBean/laravel-plausible/blob/main/resources/views/tracking.blade.php

:thinking_face:

**Comment by VincentBean** [comment]
Hi, one is a blade component and the other is a view. That way you have two ways of including the tracker, just use what you prefer.
